easybuild/easyconfigs/u/USEARCH/USEARCH-11.0.667-i86linux32.eb

```
@@ -0,0 +1,31 @@
+easyblock = 'Binary'
+
+name = 'USEARCH'
+version = '11.0.667'
+versionsuffix = '-i86linux32'
+
+homepage = 'https://www.drive5.com/usearch/index.html'
+description = """USEARCH is a unique sequence analysis tool which offers search and clustering algorithms that are
+ often orders of magnitude faster than BLAST."""
+
+toolchain = SYSTEM
+
+# download via https://www.drive5.com/usearch/download.html
+local_usearch_bin = 'usearch%s_%s' % (version, versionsuffix[1:])
+sources = [local_usearch_bin]
+checksums = ['9cc2864b1fb0f731386c92d61324a947d8a8afabe3b04f9939b9062c99fcb917']
+
+install_cmd = "mkdir -p %%(installdir)s/bin && mv %s %%(installdir)s/bin/%%(namelower)s && " % local_usearch_bin
+install_cmd += "chmod a+rx %(installdir)s/bin/%(namelower)s"
```

the licence page explicitly says:
>I agree to set execute-only permissions --x--x--x on the binary.

so, to be on the safe side, probably better to do this:
```suggestion
install_cmd += "chmod a=x %(installdir)s/bin/%(namelower)s"
```

---

Updated command to reflect suggested changes and resubmitted